boards/ek-lm4f120xl/include/periph_conf.h

```diff
@@ -99,6 +101,68 @@ extern "C" {
 #define ADC_NUMOF           (12)
 /** @} */
 
+/**
+ * @name SPI configuration
+ * @{
+ */
+static const struct spi_conf_s spi_confs[] = {
```

Comment:
I think that's the preferred way to go, if possible, as I see that the actual code relies on the vendor implementation, so I thought you made it this way because it was easier to use with the vendor interface.